moped-database/migrations/1730324794396_add_moped_user_saved_views_table/up.sql

```diff
@@ -0,0 +1,36 @@
+CREATE TABLE public.moped_user_saved_views (
+    id serial NOT NULL,
+    description text NOT NULL,
+    url text NOT NULL,
+    query_filters jsonb,
```

Comment:
@tillyw Could you update this `query_filters` column to be `NOT NULL` too? Just saw this when browsing the DB